public/app/plugins/panel/table/TablePanel.tsx

```diff
@@ -133,7 +133,7 @@ export class TablePanel extends Component<Props> {
     }
 
     if (count > 1) {
-      const inputHeight = config.theme.spacing.formInputHeight;
+      const inputHeight = config.theme2.v1.spacing.formInputHeight;
```

Comment:
minor nit: could we use `theme.components.height.md`?

Reply:
yes we could! missed this one